### ISSUE TITLE
Panic on bogus menu

### ISSUE THREAD

**Issue by KillianKemps** [KillianKemps/rs-gopher-client#5]
Marmotte panicks with this error when accessing a bogus menu: 
```
thread 'main' panicked at 'byte index 1 is out of bounds of ``', src/libcore/str/mod.rs:2051:9
```

URL of bogus Menu: gopher://khzae.net:70/1/s1000d

The bogus menu has a line without an item type.
On vf-1 I see this line is not displayed at all.

Part of the page retrieved by curl:
```
$ curl gopher://khzae.net:70/1/s1000d

iTools and other resources for the Specification 1000D		error.host	1
i 		error.host	1
		''.                  ....                            	70
i        ..,'',:,;.'''.            .'.  .'..',cc;;,;,''..             		error.host	1
i            cllll:c:.              .,,;c:,c,:cllllllllc;             		error.host	1
i              'c..              .clllllllc;l:;..;lc';lc,.            		error.host	1
[…]
```

We should avoid to have a panic error and maybe try to fallback like vf-1 by ignoring bogus lines.

**Comment by KillianKemps** [comment]
With the coming fix in #6 this part of the page will be displayed like this:
```
marmotte> go gopher://khzae.net:70/1/s1000d
		Tools and other resources for the Specification 1000D
		 
ERR		marmotte: Problem parsing line 2: Could not parse item type in: "		''.                  ....                            	70"
		        ..,'',:,;.'''.            .'.  .'..',cc;;,;,''..             
[…]
```

**Comment by KillianKemps** [comment]
PR https://github.com/KillianKemps/marmotte/pull/6 fixes this issue
